.github/workflows/bump-version.yml

```diff
@@ -20,6 +20,12 @@ on:
       new_version:
         description: 'New version to bump to'
         required: true
+        type: string
+      target_branch:
+        description: 'Branch to push tag to'
+        required: true
+        type: string
+        default: 'main'
```

Comment:
I actually think this isn't getting picked up correctly as a default, so I should check the GHA docs on this. :/

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs